### ISSUE TITLE
Placing an image onto the canvas on right - currently only click. Should be drag as well

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#1706]
Watching people working, it seems that because you drag activities into the Jig panel, they expect that placing images on the canvas is also with a drag. It is only with click at the moment. Can it be a drag as well, even if it is always to the center, as the click is now?

**Comment by dakom** [comment]
yeah- drag and drop, where on drop it functionally does what click does now (always places in center) should be pretty easy to implement

**Comment by corinnewo** [comment]
@MendyBerger Again, feedback from users in the beta testing, that it is inconsistent that you drag into the JIG, but images need to be clicked and can't be dragged. We need to add the drag!

**Comment by MendyBerger** [comment]
FYI I don't expect this feature to be easy to implement
